back/app/models/static_page.rb

```diff
@@ -0,0 +1,68 @@
+# == Schema Information
+#
+# Table name: pages
```

Comment:
Could you please run `rails db:migrate` again to update this annotation? I suppose you haven't committed it after the latest run.